InternTrack.Core.Api.Tests.Unit/Services/Foundations/Interns/InternServiceTests.Exceptions.Modify.cs

```diff
@@ -0,0 +1,235 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using InternTrack.Core.Api.Models.Interns.Exceptions;
+using InternTrack.Core.Api.Models.Interns;
+using Microsoft.Data.SqlClient;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
![image](https://github.com/hassanhabib/InternTrack.Core/assets/1453985/722c7b90-61bd-4bb4-92ba-fe0925d522c8)

Make sure that when you run code clean up that System directives are always on top